### PR TITLE
Project initialization

### PR DESCRIPTION
**Pull request by SylvG**
Project initialization with tools installation : 

- Docker
- Symfony 2
- Grunt
- Sass
- Babel

### CODE REVIEW

File: app/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+common:
+    base:
+        header:
+            site_name: 'Mo<span>W</span>lkky'
+            admin_access: 'Accès administrateurs et arbitres'
+            signin: 'S''identifier'
```
Comment (HitoTech) [inline app/Resources/translations/messages.fr.yml]: Plutôt 'S\'identifier' non ?

File: docker/engine/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM php:7.0-fpm
+
+RUN apt-get update && \
+    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```
Comment (HitoTech) [inline docker/engine/Dockerfile]: Quel est l'intérêt de l'image composer/composer si tu installe composer dans cette image ?

File: app/Resources/translations/messages.fr.yml (hunk 1 above)
Comment (SylvG) [inline app/Resources/translations/messages.fr.yml]: Not in YAML. Simple quote is escaped by ''

http://symfony.com/doc/current/components/yaml/yaml_format.html

File: docker/engine/Dockerfile (hunk 2 above)
Comment (SylvG) [inline docker/engine/Dockerfile]: I just forgot to remove it :)
I'll do it.

**Pull request merged**